vernac/g_vernac.mlg

```diff
@@ -343,7 +379,7 @@ GRAMMAR EXTEND Gram
       | -> { RecordDecl (None, []) } ] ]
   ;
   inductive_definition:
-    [ [ oc = opt_coercion; id = ident_decl; indpar = binders;
+    [ [ oc = opt_coercion; id = cumul_ident_decl; indpar = binders;
```

Comment:
How about adding the new/updated productions into the manual with some description?  `inductive_definitions` is shown in the `Inductive` command.  If you like I can push a commit with the updated grammar (or you can figure out how to use doc_grammar to do so yourself with `make doc_gram_rsts`).

Also, this change only applies to `inductive_definition` and not to the 10 other grammar references to `ident_decl`?

Reply:
> If you like I can push a commit with the updated grammar

Please do.

>Also, this change only applies to inductive_definition and not to the 10 other grammar references to ident_decl?


Correct.

Reply:
I pushed a commit with the doc/grammar updates, including a few unrelated things from master.  The way the tooling works, updating only part of the syntax is not simple.